pkg/operator/controller/controller_test.go

```diff
@@ -1459,3 +1474,148 @@ func createReconciler(client realClient.Client) *ReconcileCDI {
 
 	return r
 }
+
+func validateEvents(events chan string, match map[string]bool) {
+	// Closing the channel allows me to do non blocking reads of the channel, once the channel runs out of items the loop exits.
+	close(events)
+	for event := range events {
+		val, ok := match[event]
+		Expect(ok).To(BeTrue(), "Event [%s] was not expected", event)
+		if !val {
+			match[event] = true
+		}
+	}
+	for k, v := range match {
+		Expect(v).To(BeTrue(), "Event [%s] not observed", k)
+	}
+}
+
+func createDeleteCDIAfterReadyEventValidationMap() map[string]bool {
+	match := createReadyEventValidationMap()
+	match["Normal DeleteResourceStart Started deleting deployment cdi-deployment"] = false
+	match["Normal DeleteResourceSuccess Deleted deployment cdi-deployment successfully"] = false
+	match["Normal DeleteResourceStart Started deleting worker resources"] = false
+	match["Normal DeleteResourceSuccess Deleted worker resources successfully"] = false
+	return match
+}
+
+func createDeleteCDIEventValidationMap() map[string]bool {
+	match := createNotReadyEventValidationMap()
+	match["Normal DeleteResourceStart Started deleting deployment cdi-deployment"] = false
+	match["Normal DeleteResourceSuccess Deleted deployment cdi-deployment successfully"] = false
+	match["Normal DeleteResourceStart Started deleting worker resources"] = false
+	match["Normal DeleteResourceSuccess Deleted worker resources successfully"] = false
+	return match
+}
+
+func createErrorCdiEventValidationMap() map[string]bool {
```

Comment:
choose either Cdi or CDI for all funcs

Reply:
Good point, fixed.

Reply:
BTW: the "The Go Programming Language" book suggests all caps for acronyms `CDI` or  all lowercase `cdi`, never mix